src/rantanplan/rhymes.py

```diff
@@ -87,7 +94,14 @@ def get_clean_codes(stressed_endings, assonance=False, relaxation=False):
         if relaxation:
             relaxed_endings = []
             for syllable in stressed_ending:
-                relaxed_syllable = WEAK_VOWELS_RE.sub(r"\1", syllable, count=1)
+                relaxed_syllable = WEAK_STRONG_VOWELS_RE.sub(r"\1", syllable,
+                                                             count=1)
+                relaxed_syllable = STRONG_WEAK_VOWELS_RE.sub(r"\1",
+                                                             relaxed_syllable,
+                                                             count=1)
+                relaxed_syllable = WEAK_WEAK_VOWELS_RE.sub(r"\1",
+                                                           relaxed_syllable,
+                                                           count=1)
```

Comment:
I might need some explanation on this.
```suggestion
                relaxed_syllable = WEAK_STRONG_VOWELS_RE.sub(
                    r"\1", syllable, count=1)
                relaxed_syllable = STRONG_WEAK_VOWELS_RE.sub(
                    r"\1", relaxed_syllable, count=1)
                relaxed_syllable = WEAK_WEAK_VOWELS_RE.sub(
                    r"\1", relaxed_syllable, count=1)
```

Reply:
You mean the WEAK_WEAK_VOWELS_RE?
Can't remeber the exact example but take this for example:
```
a mil grados celsius
me frieron el lémur
```
Without that, it doen't detect the rhyme e-u

Reply:
I worry that will have unintended consequences, but it all test pass they pass. After this PR is merged we should check the accuracy on Navarro-Colorado 100 poems corpus. Maybe even add that as a pre-condition before merging into develop.

Reply:
I'm also wondering if always adding the stress to the last word will lower the accuracy, even if it's the right thing o do. Maybe it's time to create a true gold standard now we have a team of philologists. We will also have the evaluation corpus that is being manually annotated with stanza type, and revise the metrical patterns... It consists on almost 500 stanzas

Reply:
As of now, the gold standard we had mixed lexical with metrical patterns, which is very problematic. Our system is optimized for such corpus. By making this and other changes we are deviating from that approach, but that also means we need a new gold standard (the 500 stanzas corpus) were the annotation is made by experts. This requires a longer conversation and the publication of such corpus with a constructive critique on existing corpora for scansion.

Reply:
Added issue https://github.com/linhd-postdata/rantanplan/issues/92.